pkg/bpfassets/attacher/libbpf_attacher.go

```diff
@@ -331,3 +301,83 @@ func resizeArrayEntries(name string, size int) error {
 
 	return nil
 }
+
+func libbpfCollectProcessBatch(processes *bpf.BPFMap) ([]ProcessBPFMetrics, error) {
+	processesData := []ProcessBPFMetrics{}
+	var err error
+	valueSize := int(unsafe.Sizeof(ProcessBPFMetrics{}))
+	entries := MapSize / valueSize
+	isEnd := false
+	for !isEnd {
```

Comment:
```suggestion
	for isEnd := false;  !isEnd {
```

Reply:
Need another `;` for for loop syntax.

```suggestion
	for isEnd := false;  !isEnd; {
```